Parameters for the passing of the orderid and tradeid changed. Delete…

wwender opened the pull request:
Delete Order and showMyOrderDetail changed and tested against the bitcoin.de API. According to the change I also changed executeTrade but this has not been tested

peshay commented:
- Why is there an additional empty line at import codecs?
- Why are there 4 new lines in the try call starting at line 98
- there is an additional space before `newuri` which would cause an intention error on python
I hope your code is tested so far and proofed to work? As I have no test routines here and so on .. :(

wwender commented:
Sorry empty lines are just from my full copy/paste of the file where I did the changes. I corrected it that way that it is identical to the original file. 
I removed the additional space before newuri. However it did work with the additional space on my environment. I double checked that showMyOrderDetails now  also worked with the removed space. (as it should)
